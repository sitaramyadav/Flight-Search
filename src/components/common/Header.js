import React from "react";
import styled from "styled-components";

const HeaderContainer = styled.header`
  display: flex;
  justify-content: center;
  align-itmes: center;
  height: 100px;
  background-color: black;
  color: white;
`;

const CXLoyalty = styled.h1`
  margin: 32px 0 0 0;
`;

export const Header = () => {
  return (
    <HeaderContainer data-testid={"Header"}>
      <CXLoyalty>cxLoyalty</CXLoyalty>
    </HeaderContainer>
  );
};
