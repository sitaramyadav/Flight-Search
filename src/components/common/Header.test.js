import React from "react";
import { render } from "@testing-library/react";
import { Header } from "./Header";
// import "@testing-library/jest-dom/extend-expect";

describe("Header", () => {
  it("should render the child components", () => {
    const { queryByTestId } = render(<Header />);
    expect(queryByTestId("Header")).toBeInTheDocument();
    expect(queryByTestId).toMatchSnapshot();
  });
});
