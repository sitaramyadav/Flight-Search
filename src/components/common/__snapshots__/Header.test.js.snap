// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the child components 1`] = `[Function]`;
